packages/contracts-bedrock/scripts/deploy/DeploySuperchain.s.sol

```diff
@@ -359,6 +359,7 @@ contract DeploySuperchain is Script {
 
     function deployAndInitializeSuperchainConfig(DeploySuperchainInput _dsi, DeploySuperchainOutput _dso) public {
         address guardian = _dsi.guardian();
+        address upgrader = _dsi.superchainProxyAdminOwner();
```

Comment:
Non blocking, but I feel like this should match the name of the role more 1:1 to ensure clarity in configuration

Reply:
the term "Upgrade Controller" is used in the [specs](https://specs.optimism.io/protocol/stage-1.html?highlight=upgrade%20controller#roles-for-stage-1)". 

Would that work, or are you thinking more like `Proxy Admin Owner`? My concern with that is that there is a chance that Superchain Upgrades remove the Proxy Admin, so while I agree that clarity is important, I think having the name indicate the action that is authorized is preferable to indicating where the auth is managed.

Reply:
Tracked this [here](https://github.com/ethereum-optimism/optimism/issues/12861). 

Resolving as "non blocking".

Reply:
I meant `superchainProxyAdminOwner` -> `dsi.upgrader()`